packages/paste-website/gatsby-config.js

```diff
@@ -150,18 +128,6 @@ const gatsbyConfig = {
       },
     },
     'gatsby-plugin-sitemap',
-    {
-      resolve: 'gatsby-plugin-manifest',
-      options: {
-        name: 'Paste: The Design System for building Twilio customer experiences',
-        short_name: 'Paste',
-        start_url: '/',
-        background_color: '#fff',
-        theme_color: '#141c2c',
-        display: 'standalone',
-        icon: 'static/logo.svg',
-      },
-    },
```

Comment:
Note to self: double check the manifest isn't actually broken. Might need to pull this in

Reply:
question: did you get around to double checking this?

Reply:
Yup, added it back in